util/metrics/metrics.go

```diff
@@ -0,0 +1,69 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// RegisterPrometheusMetrics register all prometheus metrics with the global
+// metrics handler.
+func RegisterPrometheusMetrics() {
+	prometheus.Register(BlockImportTimeHistogramSeconds)
+	prometheus.Register(CumulativeImportTimeCounter)
+	prometheus.Register(CumulativeTransactionsCounter)
+	prometheus.Register(ImportedTransactionsPerBlockHistogram)
+	prometheus.Register(CurrentRoundGauge)
+}
+
+const (
+
+	// ImportTimePerBlockHistogramName metric name.
+	ImportTimePerBlockHistogramName = "average_import_time_sec"
+	// ImportTimeCounterName metric name.
+	ImportTimeCounterName = "cumulative_import_time_sec"
+	// TransactionsPerBlockHistogramName metric name.
+	TransactionsPerBlockHistogramName = "average_imported_tx_per_block"
+	// ImportedTransactionsCounterName metric name.
+	ImportedTransactionsCounterName = "cumulative_imported_tx"
+	// CurrentRoundGaugeName metric name.
+	CurrentRoundGaugeName = "current_round"
+)
+
+var (
+	// BlockImportTimeHistogramSeconds average block import duration in seconds.
+	BlockImportTimeHistogramSeconds = prometheus.NewSummary(
+		prometheus.SummaryOpts{
+			Subsystem: "indexer_daemon",
+			Name:      ImportTimePerBlockHistogramName,
```

Comment:
Would be nice to make the name of this variable consistent with the name of the prometheus variable. Right now there are so many variables + string constants, it is confusing.

Reply:
I tried to shorten these a bit, and make the naming a bit more consistent